scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -53,10 +52,31 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     };
     this.activeFilterTags = {};
 
+    Promise.all(
+        Object.values(extensions)
+            .map(
+                (extension) =>
+                    extension.activationResult?.contributions?.personalSpace?.getSections?.(),
+            ),
+    ).then((res) => {
+        const response = _.flattenDeep(res);
```

Comment:
It's better to only import the thing you need. `import {flattenDeep} from 'lodash'` That way bundle size is smaller, because it's not importing the entire library.